src/pages/Main.tsx

```diff
@@ -2,43 +2,56 @@ import Badges from 'components/badges/Badges'
 import CardSeparator from 'components/CardSeparator'
 import ProofsCard from 'components/proofs/Card'
 import ZkProofHint from 'components/ZkProofHint'
+import ZkProofHintWhenLg from 'components/proofs/ZkProofHintWhenLg'
 import classnames, {
   alignItems,
   display,
   flexDirection,
+  inset,
   justifyContent,
+  margin,
+  position,
 } from 'classnames/tailwind'
 import useBreakpoints from 'hooks/useBreakpoints'
 
-const mainBlock = classnames(
+const cardsContainer = classnames(
   display('flex'),
   flexDirection('flex-col', 'lg:flex-row'),
   alignItems('items-center', 'lg:items-stretch'),
-  justifyContent('lg:justify-center')
+  justifyContent('lg:justify-center'),
+  margin('mx-4')
+)
+
+const proofHintContainer = classnames(
+  position('lg:relative'),
+  inset('lg:right-1/4')
 )
 
 export default function () {
   const { lg } = useBreakpoints()
 
   return (
-    <div className={mainBlock}>
-      <ProofsCard />
-      <CardSeparator
-        numberOfLines={3}
-        gradient="accent-to-secondary"
-        vertical={!lg}
-      />
-      <Badges />
-      {!lg && (
-        <>
+    <div>
```

Comment:
I think we need to simplify this construction

Reply:
It says you marked this as resolved, was this on purpose? If not I'll work on simplifying this in the morning